apps/common-app/src/examples/ScrollToFlashListExample.tsx

```diff
@@ -0,0 +1,81 @@
+import type { AnimatedProps } from 'react-native-reanimated';
```

Comment:
It looks like scrollTo examples for `ScrollView`, `FlatList`, and `FlashList` look basically the same if you ignore the scrollable component. I wonder if it would be possible to make them one example and inject the scrollable container as a prop to the example.

Reply:
I moved these 3 examples to a single file but it is now much harder to type it properly without the usage of `any` type. Also, `forwardRef` and `FlashList` don't seem to play nice with the `AnimatedRef` so we can't go without type casting.